packages/backend/server/src/core/workspaces/management.ts

```diff
@@ -117,7 +117,7 @@ export class WorkspaceManagementResolver {
   async availableFeatures(
     @CurrentUser() user: CurrentUser
   ): Promise<FeatureType[]> {
-    const isEarlyAccessUser = await this.feature.isEarlyAccessUser(user.email);
+    const isEarlyAccessUser = await this.feature.canEarlyAccess(user.email);
```

Comment:
why use `canEarlyAccess`, this method will always true if `earlyAccessPreview` flag not enabled